tools/ci_build/github/android/build_aar_and_copy_artifacts.sh

```diff
@@ -10,25 +10,43 @@ export PATH=/opt/python/cp38-cp38/bin:$PATH
 
 ls /build
 ls /build/deps
+
+# User inputs
+USE_QNN=${1:-0}  # by default qnn will not be included in package
```

Comment:
if we ever add more options to this script, we should use arguments with a more meaningful name. e.g., `--use_qnn` instead of `1`.